libcanard/canard.c

```diff
@@ -74,8 +74,37 @@ typedef uint16_t TransferCRC;
 #define CRC_RESIDUE 0x0000U
 #define CRC_SIZE_BYTES 2U
 
+#ifdef CANARD_CRC_TABLE
```

Comment:
This is unsafe because `CANARD_CRC_TABLE=0` yields potentially confusing behavior. The safe way to handle this is to add this at the top of the file, next to the other configuration parameters:

```c
/// Docs docs docs
#ifndef CANARD_CRC_TABLE
#   define CANARD_CRC_TABLE 1
#endif
```

I think the table should be the default option because it has a huge effect on performance. I might even suggest that the table be the only option, although I won't insist.

Reply:
Agreed. I'll make it default.